test/pools/LinearLiquidityPool/TestMulticoinTrading.sol

```diff
@@ -0,0 +1,245 @@
+pragma solidity >=0.6.0;
+
+import "truffle/Assert.sol";
+import "./BaseMulticon.sol";
+
+contract TestMulticoinTrading is Base {
+    event LogUint(string, uint);//Log event
+
+    StablecoinMock stablecoinA;//Stablecoin A
+    StablecoinMock stablecoinB;//Stablecoin B
+    StablecoinMock stablecoinC;//Stablecoin C
```

Comment:
Redundant comment can be removed. The variable name is arelady descriptive enough.